### DISCUSSION TITLE
todo(todo.txt-cli, todo.txt and nostr)?

### DISCUSSION THREAD

**Discussion by ghost** [todotxt/todo.txt-cli#435] [Q&A] [answered]

**Comment by benjaminoakes** [comment, marked as answer]
I'm not entirely sure how I ended up on this thread but I'm all for avoiding a dependency on Dropbox, Google Drive, etc.

Personally, I've been using `git` to synchronize my `todo.txt`.  There used to be a project called [SparkleShare](https://www.sparkleshare.org/) that I used, but it's basically just `while true; do git commit -am "Snapshot"; git pull --rebase; git push; sleep 60; done`.  And of course, you can host your own `git` repository very simply.  I don't know if any of this can work well on iOS, but there are a few ways of making `git` play nice with Android.

I've also heard [Syncthing](https://syncthing.net/) can work well.
